pypeit/utils.py

```diff
@@ -186,10 +212,16 @@ def fast_running_median(seq, window_size):
     See discussion at:
     http://groups.google.com/group/comp.lang.python/browse_thread/thread/d0e011c87174c2d0
     """
+    # Enforce that the window_size needs to be smaller than the sequence, otherwise we get arrays of the wrong size
+    # upon return (very bad). Added by JFH. Should we print out an error here?
+
+    if (window_size > (len(seq)-1)):
+        msgs.warn('window_size > len(seq)-1. Truncating window_size to len(seq)-1, but something is probably wrong....')
```

Comment:
Yeah I agree. The reason I'm reluctant to do so, is that this code constantly causes crashes in flat fielding where it is used, because they inputs, either the slits, or a saturated flat are often junk. This then causes one bad slit resulting in the entire mask crashing. We need a global solution for this problem, but we don't have one yet.